models/src/test/scala/coop/rchain/models/blockImplicits.scala

```diff
@@ -44,7 +44,7 @@ object blockImplicits {
     validator       = ByteString.copyFrom(byteArray.toArray)
   } yield Justification(validator, latestBlockHash)
 
-  val signedDeployDataGen: Gen[Signed[DeployData]] =
+  def signedDeployDataGen(shardId: String): Gen[Signed[DeployData]] =
```

Comment:
ShardId should be also generated as other fields. It can be overwritten by the caller.
`shardId <- arbitrary[String]`

The same for other Gen functions in this file.

Reply:
[Fixed](https://github.com/rchain/rchain/pull/3631/commits/c384e9ba83a54e7e7bc910661c667c56f31e887c).